cmd/tailscale/cli/licenses.go

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2020 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package cli
+
+import (
+	"context"
+
+	"github.com/peterbourgon/ff/v3/ffcli"
+)
+
+var licensesCmd = &ffcli.Command{
+	Name:       "licenses",
+	ShortUsage: "licenses",
+	ShortHelp:  "Get open source license information",
+	LongHelp:   "Get open source license information",
+	Exec:       runLicenses,
+}
+
+func runLicenses(ctx context.Context, args []string) error {
+	outln(`
+Tailscale wouldn't be possible without the contributions of thousands of open
+source developers. To see the open source packages included in Tailscale and
+their respective license information, visit:
+
+    https://tailscale.com/licenses/tailscale`)
```

Comment:
macOS systems and Windows systems can also run the `tailscale licenses` command (on macOS via `/Applications/Tailscale.app/Contents/MacOS/Tailscale licenses`). Should we check GOOS?

Tailscaled-on-MacOS would be a corner case, its GOOS=darwin but its actual dependencies would not be the same as licenses/apple

Reply:
https://tailscale.com/licenses/tailscale includes the dependencies for the CLI for all architectures, so it will still be the right link.  Which is why https://tailscale.com/licenses/apple additionally points to the CLI page at the top, since we do distribute the CLI with it.  This is the 80% solution that ensures we're listing all of our dependencies, even if the grouping is a tad weird.

Reply:
So it would be correct for the CLI on all platforms to point to https://tailscale.com/licenses/tailscale, and then the system tray app on macOS will point to https://tailscale.com/licenses/apple

Reply:
I'll admit to possibly overthinking this.

`/Applications/Tailscale.app/Contents/MacOS/Tailscale.app` links to `wireguard-apple` and uses it in operation.

`wireguard-apple` will not appear in `/Applications/Tailscale.app/Contents/MacOS/Tailscale.app licenses` because it is not in `tailscale.md`, because it doesn't appear in the OSS repo.

macOS only has to one giant app bundle, not separate tailscaled and tailscale binaries.

Reply:
ah, got it.  I misunderstood that part.  And not overthinking it... I'd like to get this right.  I'll add in a GOOS check